pages/property-filter/count-text.page.tsx

```diff
@@ -0,0 +1,142 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Added this test page for manually testing the changes with the different screen readers.